src/utils/FetchConfig.ts

```diff
@@ -94,6 +94,9 @@ export async function FetchConfig (): Promise<any> {
     console.info('Set Siteminder Logout URL to: ' + siteminderLogoutUrl)
   }
 
+  const hotjarId: string = response.data.HOTJAR_ID;
+  (<any>window).hotjarId = hotjarId
```

Comment:
This is to load the ID from the local config file (when run locally) or from the fetched config file (when run in Dev/Test/Prod).